scripts/manifests/dev.json

```diff
@@ -32,7 +32,7 @@
     "webRequest",
     "webRequestBlocking",
```

Comment:
I guess the blocking permission should no longer be required?

Reply:
That is correct, please then also remove it:
```suggestion
```